ResourceType.dhall

```diff
@@ -0,0 +1,50 @@
+let ResourceType
+    : ∀(_params : { Source : Type, SourceGetParams : Type }) →
+        { Type : Type
+        , default :
+            { privileged : Optional Bool
+            , params : Optional _params.SourceGetParams
+            , check_every : Optional Text
+            , tags : Optional (List Text)
+            , unique_version_history : Optional Bool
+            }
+        }
+    = λ(_params : { Source : Type, SourceGetParams : Type }) →
+        { Type =
+            { name : Text
+            , type : Text
+            , source : _params.Source
+            , privileged : Optional Bool
+            , params : Optional _params.SourceGetParams
+            , check_every : Optional Text
+            , tags : Optional (List Text)
+            , unique_version_history : Optional Bool
+            }
+        , default =
+          { privileged = None Bool
+          , params = None _params.SourceGetParams
+          , check_every = None Text
```

Comment:
Shouldn't the default be 24h?

Reply:
@oded-dd The upstream field is optional... I can imagine more than a number of cases where we can't use webhooks to trigger checking the resource and as such benefit from a shorter check_every